c/zosaccounts.c

```diff
@@ -220,6 +322,11 @@ int groupInfoGetGroupId (GroupInfo *info) {
   return groupId;
 }
 
+/* Copy group name from Group Info structure into groupNameBuffer*/
+void groupInfoGetGroupName (GroupInfo *info, char *groupNameBuffer) {
```

Comment:
Necessary to have its own method?

Reply:
It's just for the sake of consistency with the existing code that provides convenient access methods e.g. `groupInfo GetGroupId`